bridge/setu/processor/clerk.go

```diff
@@ -88,6 +99,13 @@ func (cp *ClerkProcessor) sendStateSyncedToHeimdall(eventName string, logBytes s
 			)
 			return nil
 		}
+		tracing.EndSpan(isOldTxSpan)
```

Comment:
We'd need to end span in case of errors as well. If `isOldTx` returns an error, we would just exit the function. 

Maybe just do 
```
_, isOldTxSpan := tracing.StartSpan(sendStateSyncedToHeimdallCtx, "isOldTx")

isOld, err := cp.isOldTx(...);

tracing.EndSpan(isOldTxSpan)

if err != nil {...}
```

Same for `checkTxAgainstMempool`. 

Also, it would be good to add an `err` attribute printing the error if any.